### DISCUSSION TITLE
Site Unaccessible from certain countries

### DISCUSSION THREAD

**Discussion by Varsilias** [vercel/community#6201] [Help] [answered]
### Summary

I have a site hosted on Vercel with a Domain purchased from Namecheap, At the time of initial deployment (last year), everything worked perfectly, The Domain was pointed perfectly via CNAME and Nameserver records.

Within the past 3 days, I have noticed that my site is inaccessible from **Nigeria** which was never the case in the past, I have cleared my browser cache and tried with different other browsers but the problem persists.

I tried accessing the site over a VPN and it worked, I could access the site from a different location. 

I have contacted Namecheap support to verify if the problem is from their end, but they have confirmed that everything looks great from their end. I have attached a screenshot of the mail to this summary
<img width="1130" alt="Screenshot 2024-03-21 at 11 14 34 AM" src="https://github.com/vercel/community/assets/61119104/67b443ea-992f-465a-83c8-d0f8ebfa0bcf">

The Domain in question is: https://danielokoronkwo.tech

### Example

https://danielokoronkwo.tech

### Steps to Reproduce

1. Visit the site from your home country [https://danielokoronkwo.tech](https://danielokoronkwo.tech)
2. If you have a VPN, switch it to Nigeria and Visit the site again

**Comment by uzoway** [comment, marked as answer]
Hi @Varsilias , I've been having the same issue. I just figured out that the problem is with MTN & Airtel recently blocking some IP Addresses and Vercel's happens to be one of them. I'm still looking for a solution aside from using a VPN.
I just followed you on Twitter should incase you find something.

**Comment by Varsilias** [comment]
Hi @uzoway thank you so much for this information, let me try and see how I can get in touch with anyone who works at either MTN or Airtel

**Reply by uzoway** [reply]
Alright!
All the best with that.

**Comment by Zonalds** [comment]
Yeap, I can confirm this is the case. Vercel sites are inaccessible in Nigeria. Our staff can use VPN but our customers wouldn't.

**Comment by jocosageDN** [comment]
I have also been having the same issue, it really sucks as it's not just from Nigeria that this issue exist from.
Is there an alternative way to keep using vercel?
I would really like to continue with Vercel.

**Comment by 32bitsret** [comment]
i came to see i wasn't the only one experiencing this issue

**Comment by akojif** [comment]
Hi everyone, it seems you've been experiencing difficulties with site accessibility on Vercel when using Nigerian Internet Service Providers. I've written a guide on how to resolve this. You can check out the article [here](https://medium.com/@akfdev/fixing-vercel-site-accessibility-issues-on-nigerian-networks-with-cloudflare-dns-f15311e1d2f6)

[a comment by an automated account is not shown]
